spec/defines/resource_server_spec.rb

```diff
@@ -904,18 +904,6 @@
         it { is_expected.to contain_nginx__resource__location("#{title}-default").with_ssl_only(true) }
       end
 
-      context 'SSL cert missing' do
```

Comment:
*if* there are other tests.  I don't have massively strong feelings though.  It was more devil's advocate.  They probably are more trouble than they're worth.